zos_subsystems/ims/ims_provisioning_wazi/test.yml

```diff
@@ -0,0 +1,250 @@
+---
```

Comment:
Is this test playbook for internal testing or actually part of the sample?

Reply:
it can be used by end users for testing purposes